ReadingList/ReadingListBatchRecordResponse.swift

```diff
@@ -7,19 +7,18 @@ import Foundation
 class ReadingListBatchRecordResponse: ReadingListResponse {
     var responses: [ReadingListRecordResponse] = [ReadingListRecordResponse]()
 
-    override init?(response: HTTPURLResponse, json: AnyObject?) {
+    override init?(response: HTTPURLResponse, json: [String: Any]) {
```

Comment:
Just a thought. Is it worth us typealiasing this?
`typealias JSONObject = [String: Any]`

This would make all references here to be very explicit, and if we have this problem again in a future migration, all we have to do is change the typealias definition to fix it.